packages/react-ai/src/components/AIConversation/views/Controls/FormControl.tsx

```diff
@@ -213,14 +221,52 @@ export const FormControl: FormControl = () => {
     }
   };
 
+  const onValidate = React.useCallback(
+    async (files: File[]) => {
+      const previousFiles = input?.files ?? [];
+      const { acceptedFiles, hasMaxError, hasMaxSizeError } =
+        await attachmentsValidator({
+          files: [...files, ...previousFiles],
+          maxAttachments,
+          maxAttachmentSize,
+        });
+
+      if (hasMaxError ?? hasMaxSizeError) {
+        let error = '';
+        if (hasMaxError) {
+          error += displayText.getMaxAttachmentErrorText(maxAttachments);
+        }
+        if (hasMaxSizeError) {
+          error += displayText.getAttachmentSizeErrorText(
```

Comment:
If both errors are present, is the string just joined together without a space in between? Would it be better to do something like
```ts
const errors = [];
errors.push(maxAttachmentError);
errors.push(sizeError);
setError(errors.join(' '));
```